src/data/item-desc.csv

```diff
@@ -69,3 +69,4 @@
 "サークルの団結の証。\nショップで様々なものと交換できる。","<size=22>Circle Points exchangeable for items at the shop.</size>"
 "★2以上確定メイクデビューガチャが引ける\nチケット。ガチャの一覧から使用できる。","<size=22>A ★2+ guaranteed character gacha pull ticket.\nUsable at the gacha menu.</size>"
 "SR以上確定メイクデビューガチャが引ける\nチケット。ガチャの一覧から使用できる。","<size=22>A SR+ guaranteed support card gacha pull ticket.\nUsable at the gacha menu.</size>"
+"タウラス杯のエントリーに\n使用できるチケット。\n※次回開催に持ち越すことはできません。","<size=22>An entry ticket for the Taurus Hai.\nOnly usable in this period.</size>"
```

Comment:
One more thing: In the part on the 2nd line (「※次回開催に持ち越すことはできません」), i feel like it would be better not to use references to the "current" period, but phrase it generally like the original japanese does.

Literally translated it would be closer to "Does not carry over between race periods.", which is roughly equal length to the japanese.

Reply:
sounds good to me, couldnt think of anything similar at the time lol